frontend/src/components/GameForm/GameForm.vue

```diff
@@ -28,8 +28,7 @@ export default Vue.extend({
 });
 </script>
 
-<!-- Add "scoped" attribute to limit CSS to this component only -->
-<style lang="sass">
+<style lang="sass" scoped>
```

Comment:
`scoped` shouldn't be required as `#add` should be unique. Also `scoped` creates ugly DOM nodes.